tf_agents/networks/multi_categorical_projection_network_test.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for tf_agents.networks.multi_categorical_projection_network."""
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from tf_agents.networks import multi_categorical_projection_network
+from tf_agents.specs import tensor_spec
+
+
+def _get_inputs(batch_size, num_input_dims):
+  return tf.random.uniform([batch_size, num_input_dims])
+
+class MultiCategoricalProjectionNetworkTest(tf.test.TestCase):
+
+  def testBuild(self):
+    output_spec = [
+      tensor_spec.BoundedTensorSpec([], tf.int32, 0, 1),
+      tensor_spec.BoundedTensorSpec([], tf.int32, 0, 4)]
+    network = multi_categorical_projection_network.MultiCategoricalProjectionNetwork(
```

Comment:
too long of a name, can you make it shorter multi_categorical_network.MultiCategoricalNetwork